src/main/resources/db/migration/V1_16__add_court_house_to_case_table.sql

```diff
@@ -0,0 +1,6 @@
+alter table darts.moj_case add moj_cth_id integer;
```

Comment:
These have the schema prefix `darts` which might cause a problem?
We haven't been using that so far, but maybe it's ok to use now.
I think this should just be pushed up into V1_13 to correct the schema, as we've went back to touch that script anyway and we seem to be blowing away our Docker volumes at the moment as these scripts may be short lived, if we go with a full schema baseline instead in S6+